app/weeklyPicks/WeeklyPicks.tsx

```diff
@@ -119,7 +119,13 @@ const WeeklyPicks = ({ NFLTeams, currentGameWeek }: Props): JSX.Element => {
     });
 
     setUserPick(userPickData);
-  }, [user.id, gameCurrentWeek.id, NFLTeams, userPick]);
+  }, [
+    user.id,
```

Comment:
Why are there more dependencies being added?

Reply:
I was updating it as per the warning from eslint

![CleanShot 2024-06-18Code001223@2x](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/d420a236-989e-43c9-9eed-9a4bc17f0c5c)

Changes reverted in latest commit.

Reply:
Only keep the dependencies that are actually needed to watch for rehydration.